specs/codecs/wac/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Specs: WAC"
+navTitle: "WAC"
+---
+
+WAC
```

Comment:
Initially named for `WebAssembly Codec` because I didn't know what this was going to look like when I started but I knew what it was for. It's turned out to be something probably more general than just for transporting data to/from WASM.

If someone else has got a name suggestion for the codec I'm game. Only rules are it cannot contain the prefix `dag-` 🙃.

Reply:
+1 to no `dag-`, I would have tried to veto that anyway!

You could use "SimpleDAG", with this just being an evolutionary step of it? Mikeal might be OK with that since we didn't push SimpleDAG far enough to be used anywhere in practice.

no other suggestions from me, naming is hard